package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "prometheus-api-metrics",
-  "version": "0.0.0",
+  "version": "3.2.0",
```

Comment:
I found this to be the problem with package-lock.json file, giving me a version difference. I restored the `3.2.0` version to the package.json

Also I had to use Node v15 with the v2 lockfileversion otherwise it would try to restore the v1 config, rewriting most of the package-lock.json

Reply:
the version should remain 0.0.0 as we use `semantic-release` to generate the next version and sets it in the package.json file

Reply:
Ah okay, thanks for explaining this. This is easy for me to revert, will switch it back

Reply:
I have reverted the package version to `0.0.0`